service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -388,14 +388,14 @@ spec:
       install:
         storage:
           type: Memory # or Elasticsearch for production mode
-        memory:
-          maxTraces: 100000
-        elasticsearch: # the following values only apply if storage:type:=Elasticsearch
-          storage: # specific storageclass configuration for the Jaeger Elasticsearch (optional)
-            size: "100G"
-            storageClassName: "storageclass"
-          nodeCount: 3
-          redundancyPolicy: SingleRedundancy
+          memory:
+            maxTraces: 100000
+            elasticsearch: # the following values only apply if storage:type:=Elasticsearch
```

Comment:
@JStickler `memory` and `elasticsearch` tags should be under `storage` as immediate child.
in the new change looks like `elasticsearch` moved under `memory`?

Reply:
Thank you Jeeva, this is exactly why I wanted a second look at this.  I've (hopefully) fixed the nesting now.